esp-hal/src/dma/pdma.rs

```diff
@@ -42,17 +42,6 @@ macro_rules! ImplSpiChannel {
             }
 
             impl RegisterAccess for [<Spi $num DmaChannel>] {
-                fn init_channel() {
-                    // (only) on ESP32 we need to configure DPORT for the SPI DMA channels
-                    #[cfg(esp32)]
-                    {
-                        let dport = unsafe { &*crate::peripherals::DPORT::PTR };
-                        dport
-                            .spi_dma_chan_sel()
-                            .modify(|_, w| unsafe { w.[< spi $num _dma_chan_sel>]().bits($num - 1) });
-                    }
-                }
```

Comment:
Where did you move this to?

Reply:
Somewhere that I then deleted, it seems XD